statistics/feedback.go

```diff
@@ -493,3 +493,127 @@ func buildNewHistogram(h *Histogram, buckets []bucket) *Histogram {
 	}
 	return hist
 }
+
+// QueryFeedbackPB is used to serialize the QueryFeedback.
+type QueryFeedbackPB struct {
```

Comment:
The name has `PB` but It's not `protobuf`.
Do you plan to change it to `protobuf` in the future?